torch/ao/quantization/experimental/_gpu_quantization/test.py

```diff
@@ -0,0 +1,1024 @@
+# mypy: ignore-errors
```

Comment:
Please move tests under the toplevel test folder

Reply:
OK